src/nupic/ntypes/ArrayBase.hpp

```diff
@@ -41,15 +41,16 @@
 #include <iostream> // for ostream, istream
 #include <string>
 #include <memory>	// for shared_ptr
+#include <vector>
 
 #include <nupic/types/Serializable.hpp>
-#include <nupic/types/c_array.hpp>
 
 #include <nupic/types/Types.hpp>
 #include <nupic/types/Sdr.hpp>
+#include <cereal/types/vector.hpp>
 
 
-
+using namespace nupic::sdr;
```

Comment:
put inside the namespace, and ideally dont import the whole ns, but just the needed part (sdr::SDR?)